src/OrchardCore.Modules/OrchardCore.Markdown/ResourceManagementOptionsConfiguration.cs

```diff
@@ -13,10 +13,7 @@ static ResourceManagementOptionsConfiguration()
 
         _manifest
             .DefineScript("easymde")
-            .SetUrl(
-                "~/OrchardCore.Markdown/Scripts/easymde.min.js",
-                "~/OrchardCore.Markdown/Scripts/easymde.js"
```

Comment:
Why has the debug version been removed?

Reply:
because that file does not exists locally.